tests/meteor.yml

```diff
@@ -0,0 +1,31 @@
+---
+runner: command
+dir: meteor
+args:
```

Comment:
I'm not sure why we don't call each file individually rather than calling 4 of them at once. It seems simpler that way because less change in the go code. Perhaps the way implemented is how it should be done in dance to report result? :thinking:

Reply:
But then we'd have X individual YAML files where X is the number of subtests and our CI would have a lot more jobs. And I suspect that we'll be adding a lot more of these in the future.

(mongodb, meteor-doc-fetcher)
(mongodb, meteor-oplog-foo)
(mongodb, meteor-oplog-bar)
(mongodb, meteor-oplog-baz)
(mongodb, meteor-oplog-x)
(mongodb, meteor-oplog-y)
(mongodb, meteor-oplog-z)
.
.
.

Reply:
> Perhaps the way implemented is how it should be done in dance to report result?

Perhaps you're right. Let me make the other changes first and then we can decide.

Reply:
Actually, it's a good idea to group subtests and run them in a separate job to reduce the overall completion time. For now I think it's fine but as is, but in the future we can scale it this way.